src/Cli/Microsoft.DotNet.Cli.Utils/FrameworkDependencyFile.cs

```diff
@@ -62,22 +62,16 @@ internal static bool TryGetMostFitRuntimeIdentifier(
 
         if (!string.IsNullOrEmpty(currentRuntimeIdentifier))
         {
-            runtimeFallbacksCandidates =
-                runtimeGraph
-                .Where(g => string.Equals(g.Runtime, currentRuntimeIdentifier, StringComparison.OrdinalIgnoreCase))
-                .ToArray();
+            runtimeFallbacksCandidates = [.. runtimeGraph.Where(g => string.Equals(g.Runtime, currentRuntimeIdentifier, StringComparison.OrdinalIgnoreCase))];
         }
         else
         {
-            runtimeFallbacksCandidates = Array.Empty<RuntimeFallbacks>();
+            runtimeFallbacksCandidates = [];
```

Comment:
I think this one is technically less performant?

Reply:
I don't think I'm concerned about empty array initialization performance overall.